### PR TITLE
Add operation tests of CopyBufferToBuffer

### PR DESCRIPTION
**Pull request by Jiawei-Shao**

### CODE REVIEW

File: src/webgpu/api/operation/command_buffer/copies.spec.ts [hunk 1]
```diff
@@ -1,33 +1,131 @@
...
+  .params(
+    params()
+      .combine(poptions('srcOffset', [0, 4, 8, 16]))
+      .combine(poptions('dstOffset', [0, 4, 8, 16]))
+      .combine(poptions('copySize', [0, 4, 8, 16]))
+  )
+  .fn(async t => {
+    const { srcOffset, dstOffset, copySize } = t.params;
+
+    const kSrcBufferSizes = [srcOffset + copySize, srcOffset + copySize + 8];
```
Comment (Kangz) [inline src/webgpu/api/operation/command_buffer/copies.spec.ts:38]: Maybe these should be in the parametrization so it's clear we iterate over them as well.

Comment (Jiawei-Shao) [reply src/webgpu/api/operation/command_buffer/copies.spec.ts:38]: I just don't want to test invalid parameters here. For example, srcBufferSize == 8 is not legal for the cases that "srcOffset + copySize > 8". Computing buffer sizes in this way can ensure the buffer size is legal in all the tests. What do you think?

Comment (Kangz) [reply src/webgpu/api/operation/command_buffer/copies.spec.ts:38]: Maybe there's a way to put the computations in the params list? But if that's impractical let's ignore this comment.

Comment (kainino0x) [reply src/webgpu/api/operation/command_buffer/copies.spec.ts:38]: You can do something like this in the .combine()-chain:

```
      .expand(p => poptions('srcBufferSizes', [p.srcOffset + p.copySize, p.srcOffset + p.copySize + 8]))
      .expand(p => poptions('dstBufferSizes', [p.dstOffset + p.copySize, p.dstOffset + p.copySize + 8]))
```

Comment (Jiawei-Shao) [reply src/webgpu/api/operation/command_buffer/copies.spec.ts]: Done. Thanks @kainino0x  for your suggestions!

**Review by Kangz — APPROVED** [review]

**Review by kainino0x — APPROVED** [review]
LGTM!

**Pull request merged**